AppHandling/Start-NavContainerAppDataUpgrade.ps1

```diff
@@ -18,39 +18,59 @@ function  Start-BcContainerAppDataUpgrade {
     Param (
         [string] $containerName = $bcContainerHelperConfig.defaultContainerName,
         [string] $tenant = "default",
-        [Parameter(Mandatory=$true)]
+        [Parameter(Mandatory = $true)]
         [string] $appName,
         [string] $appVersion = "",
-        [string] $language = ""
+        [string] $language = "",

```

Comment:
path?
and what is path? if that should be a path specification, should it be a specification inside the container or outside?
Use Get-BcContainerPath -containerName -path to get the shared path of a path on the host if this is what's needed.

Reply:
You are right I forgot that I am in conatiner

Reply:
I tried change path like Publish-BCContainerApp.ps1, but I am not sure if it is ok this way.

Reply:
Most places where we have this - we just use

```
Invoke-ScriptInBcContainer -containername $containername -scriptblock { Param($containerPath)

code

} -argumentList (Get-BcContainerPath -containerName $containerName -path $path -throw)
```

Then people will need to place the files in a location, which is shared with the container - but that can easily be done by a simple copy to the container - or share the folder with the container.

I think we should do this

Reply:
Please check if it is OK.